packages/icons/README.md

```diff
@@ -16,6 +17,14 @@ npm install @vibe/icons
 import { Close } from "@vibe/icons";
```

Comment:
Maybe we should state here that this path will serve React components, same as you did for the raw one

Reply:
Added